diff --git a/toolkit/components/extensions/ExtensionActions.sys.mjs b/toolkit/components/extensions/ExtensionActions.sys.mjs
index 7aa620587a2fdd5a0c39cdccaf02c90ecded38fb..e2708711e07ad39af56c772c7f67ea47225f7796 100644
--- a/toolkit/components/extensions/ExtensionActions.sys.mjs
+++ b/toolkit/components/extensions/ExtensionActions.sys.mjs
@@ -215,10 +215,10 @@ class PanelActionBase {
       return;
     }
     this.activeTabForPreload = tab;
-    if (tab) {
+    if (tab && !tab.hasAttribute("zen-empty-tab")) {
       this.extension.tabManager.addActiveTabPermission(tab);
     }
-    if (oldTab) {
+    if (oldTab && !oldTab.hasAttribute("zen-empty-tab")) {
       this.extension.tabManager.revokeActiveTabPermission(oldTab);
     }
   }
@@ -239,6 +239,10 @@ class PanelActionBase {
       return null;
     }
 
+    if (tab.hasAttribute("zen-empty-tab")) {
+      return null;
+    }
+
     // Now that the action is actually being triggered we can clear any
     // existing preloaded activeTab permission.
     this.setActiveTabForPreload(null);
